shared/components/NextCommunityEvent/index.js

```diff
@@ -74,44 +86,50 @@ class NextCommunityEvent extends React.Component {
       featuredEventDescription,
       eventId,
     } = this.props;
+
+    if (eventId && !title) {
```

Comment:
shouldn't this be handled in the main router for pages that cant be found ?

Reply:
Yea I tried that but I couldn't figure out how to do it, wanna give it a look?